tmt/steps/discover/shell.py

```diff
@@ -288,7 +298,10 @@ def fetch_remote_repository(
             self.run(Command('git', 'checkout', '-f', str(ref)), cwd=testdir)
 
         # Remove .git so that it's not copied to the SUT
-        shutil.rmtree(os.path.join(testdir, '.git'))
+        # if 'sync-repo' option is not specified
+        requires_git = self.opt('sync-repo')
+        if not requires_git:
```

Comment:
What about syncing the` .git` repo by default? Is there any specific reason to do it in the opposite way?

Reply:
The reason to skip `.git` directory was to minimize the amount of data which needs to be synced to the guest.